aws-s3-transfer-manager/src/middleware/hedge.rs

```diff
@@ -37,16 +37,6 @@ pub(crate) struct Builder<P> {
 #[derive(Debug, Clone, Default)]
 pub(crate) struct DefaultPolicy;
 
-impl<T: Clone> Policy<T> for DefaultPolicy {
```

Comment:
If we aren't using `DefaultPolicy` anymore remove it and if we are we probably want to leave this trait implementation in place or else it can't function as intended.

Reply:
we are still using `DefaultPolicy`, the only implementation we need is for `UploadPartRequest` since we only do hedging for upload.